src/cbc_sdk/helpers.py

```diff
@@ -111,7 +111,7 @@ def get_object_by_name_or_id(cb, cls, name_field="name", id=None, name=None, for
     try:
         if id:
             attempted_to_find = "ID of {0:d}".format(id)
-            objs = [cb.select(cls, id, force_init=force_init)]
+            objs = [cb.select(cls, id)]
         else:
```

Comment:
This function requires either an id or name we should through an error neither are provided. I would leave the `name=None` then convert the if else to an if elif else where the else raises an Exception indicating that either id or name should be provided. We should also consider the case where both are provided, do we want to fetch by id then verify if the name matches or raise an error before making an API request indicating only one can be specified